usecases/aggregate_node_data.py

```diff
@@ -0,0 +1,21 @@
+from typing import Union
+
+from deepdiff import DeepDiff
+from gateways.node_gateway import NodeGateway
+
+
+class AggregateNodeData:
+
+    def __init__(self, node_gateway: Union[NodeGateway, None] = None) -> None:
+        self.node_gateway = node_gateway or NodeGateway()
+
+    def aggregate(self) -> Union[bool, None]:
```

Comment:
Missing docstring. What's the difference from returning a bool or a None?

Reply:
`StrictRedis set` has bool or None as returning type. It's just to match that. We can probably abstract that to only bool. I just didn't think about it.